Resources/Documentation/settings.md

```diff
@@ -6,6 +6,8 @@ Run edit mode by typing:
 ```
 vrecord -e
 ```
+(https://drive.google.com/open?id=1BY6Mu2qkBg1l7oETNBsFNK8Qs_hU6f-k)
```

Comment:
Were you trying to insert an image or a link?

Reply:
Updated to image - I think resources should be kept as local as possible

Reply:
my bad! Yes, it was supposed to be an image.

Reply:
![Screen Shot 2020-01-21 at 10 59 52 AM](https://user-images.githubusercontent.com/13877631/72834217-32224600-3c3d-11ea-8d1d-1897ab7a1975.png)